docs_src/use-cases/use-cases.md

```diff
@@ -1,4 +1,7 @@
 # Intel Retail Use Cases
 
 [Automated Self Checkout](./automated-self-checkout/automated-self-checkout.md)
-[AI Connect for Scientific Data (AiCSD)](./AiCSD/aicsd.md)
\ No newline at end of file
+
+[AI Connect for Scientific Data (AiCSD)](./AiCSD/aicsd.md)
+
+[Loss Prevention](./loss-prevention/loss-prevention.md)
```

Comment:
i am changing this too in my PR, and based on Neethu's feedback we should add the bullet points to have a better view in markdown. So let's consolidate this and check out my PR there: https://github.com/intel-retail/documentation/pull/36/files  for use-cases.md

Reply:
@jim-wang-intel can you add the "Object Detection with PyTorch and YOLO" as a bullet point in your PR ? I can avoid opening another PR for this small change

Reply:
i don't have your md file @NeethuES-intel for the bullet you want to add- i can add a place holder bullet point for that but no md file available for your PR.

Reply:
Okay, I will resolve this to comply with yours to avoid conflicts

Reply:
thank you Sean ! 👍

Reply:
> i don't have your md file @NeethuES-intel for the bullet you want to add- i can add a place holder bullet point for that but no md file available for your PR.

Ah ok. Never mind, don't include since its part of Automated Self Checkout.